react/AppIcon/index.jsx

```diff
@@ -107,6 +114,8 @@ AppIcon.propTypes = {
   alt: PropTypes.string,
   /** Required if fetchIcon is not provided */
   app: PropTypes.oneOfType([AppDoctype, PropTypes.string]),
+  /** Icon to fallback on error (optional), default cube icon */
+  fallbackIcon: PropTypes.string,
```

Comment:
is it a string or an IconProptypes ? You have access to iconPropType from `Icon`

Reply:
I didn't see that, indeed, I add it 👍